Cargo.toml

```diff
@@ -22,6 +22,7 @@ members = ["testcrate"]
 futures-await-async-macro = { path = "futures-await-async-macro", version = "0.1" }
 futures-await-await-macro = { path = "futures-await-await-macro", version = "0.1" }
 futures = "0.1"
+futures-cpupool = "0.1"
```

Comment:
Could this be moved to `[dev-dependencies]` as I think it's not necessary for the crate itself?